packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -135,41 +117,26 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, any> = {
           color: circularColor,
           backgroundColor: circularBackgroundColor,
           borderColor: circularBorderColor,
-          borderRadius: circularRadius,
+          borderRadius: circularBorderRadius,
+
           ':hover': {
             color: circularColorActive,
             backgroundColor: circularBackgroundColorHover,
             borderColor: circularBorderColorHover,
           },
-          ...(isFromKeyboard && {
-            ':focus': {
-              color: circularColorActive,
-              backgroundColor: circularBackgroundColorFocus,
-              borderColor: circularBorderColorFocus,
-              ':after': {
-                content: '""',
-                position: 'absolute',
-                top: '0',
-                right: '0',
-                bottom: '0',
-                left: '0',
-                borderWidth: `${pxToRem(borderWidth)}`,
-                borderStyle: 'solid',
-                borderColor: `${circularBorderColorFocusIndicator}`,
-                borderRadius: circularRadius,
-              },
-            },
-          }),
-          ...(!isFromKeyboard && {
-            ':focus': {
-              ':active': {
+
+          ':focus': isFromKeyboard
+            ? {
                 color: circularColorActive,
-                backgroundColor: circularBackgroundColorActive,
-                borderColor: circularBorderColorActive,
-                boxShadow: 'none',
```

Comment:
shouldn't `boxShadow: none` be preserved, the same way it is done for regular one here?

https://github.com/stardust-ui/react/pull/1269/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-62f5a77adef0b99461fbb8c43b4c5b04R101

Reply:
great catch, thanks, fixed